com.unity.render-pipelines.universal/Runtime/2D/Light2D.cs

```diff
@@ -144,22 +155,35 @@ internal int GetTopMostLitLayer()
                 return -1;
         }
 
-        internal void UpdateMesh()
+        internal void UpdateMesh(bool useCachedMesh)
```

Comment:
this method has the inverted of the common pattern of `Update(bool force)` let's use that so we don't get boolean confused

Reply:
Fixed in https://github.com/Unity-Technologies/Graphics/pull/2230/commits/8e860bf31168ba420beab74f12059ab539300a3c